client/src/Page/HallOfFame.tsx

```diff
@@ -0,0 +1,19 @@
+import FlavorThumbnail, { Flavor } from "../components/FlavorThumbnail";
+import styles from "./hall-of-fame.module.css"
+
+
+export default function HallOfFame() {
+    const flavors: Flavor[]  = Array.from( {length: 22}, () =>  ({
```

Comment:
would be good to declare this outside of the component so it doesn't get recreated every time the component re-renders.

Reply:
Good call. I put it inside b/c it'll get replaced eventually by data passed in or fetched via API, but that's a good reminder to build good habits!